test/elixre_test.exs

```diff
@@ -25,4 +25,14 @@ defmodule ElixreTest do
     assert conn.status == 404
     assert conn.resp_body == "Page not found"
   end
+
+  test "/regex" do
+    conn =
+      :post
+      |> conn("/regex")
```

Comment:
You're not sending any JSON here, you've got to send it as the body of your request :)